app/src/main/AndroidManifest.xml

```diff
@@ -60,10 +59,10 @@
         </activity>
         <activity
             android:name=".UserSettingsActivity"
+            android:configChanges="orientation|screenSize"
             android:label="@string/user_settings_activity"
             android:parentActivityName=".MainActivity"
-            android:windowSoftInputMode="adjustPan|stateAlwaysHidden"
-            android:configChanges="orientation|screenSize">
```

Comment:
I apologise if I misunderstood something but I think the config changes are there. I just reformatted the code of "AndroidManifest" since it was not aligned properly.